### ISSUE TITLE
build failed with cfg

### ISSUE THREAD

**Issue by baoyachi** [sagebind/isahc#272]
error detail:
```bash
error: attributes are not yet allowed on `if` expressions
   --> /root/.cargo/registry/src/isahc-0.9.14/src/config/dial.rs:137:9
    |
137 |         #[cfg(unix)]
    |         ^^^^^^^^^^^^

error: aborting due to previous error
```

**Comment by sagebind** [comment]
You will likely need Rust 1.45 or newer to compile. The 0.9 series only guarantees compilation on the latest Rust versions, though the upcoming 1.0 version has an MSRV of Rust 1.41.

I probably _could_ backport the changes to make Isahc compatible with 1.41 to 0.9.x, but I'm not sure it is worth it at this point since 1.0 is so close to release.

See also https://github.com/sagebind/isahc/issues/219.

**Comment by baoyachi** [comment]
Yeah,I solved  in version `isahc = "1.0.0-beta.1"`.

In addition, I would like to ask about when the official release 1.0 will take place. Any roadmap list?

**Comment by sagebind** [comment]
It is scheduled before the end of the month. See the tracking issue https://github.com/sagebind/isahc/issues/25 and watch for an announcement on the [announcements board](https://github.com/sagebind/isahc/discussions?discussions_q=category%3AAnnouncements). If you have any feedback after using the beta, I'd love to hear about it on the discussions page!

That said, if this is for a production project right _now_, I'd advise to use 0.9.x and simply updgrade Rust versions.
